scale-ci-uperf/sshd.sh

```diff
@@ -26,6 +26,9 @@ echo "PidFile /tmp/uperf_ssh/sshd.pid" >> /tmp/uperf_ssh/sshd_config
 touch /tmp/sshd.log
 /usr/sbin/sshd -E /tmp/sshd.log -f /tmp/uperf_ssh/sshd_config -D &
 
+echo "Setting net.ipv4.ip_local_port_range to control ports that uperf will use."
```

Comment:
Since you're using a privileged container, you're changing a node-level sysctl.  I believe there should definitely be a roll-back of this sysctl for other tests (this would definitely affect HTTP tests).  Node tuning operator comes to mind that will a) set the correct sysctls on a node based on a specific pod label and b) do a rollback of these settings (in 4.2.x) automatically once the pod with the label terminates.  Over this solution, the node tuning operator has the disadvantage of taking a while to actually apply the new node-level settings, rollback not being present in 4.1x and could be an overkill for this specific case, but a rollback of some sort to previous values is very likely needed.

Reply:
Interesting, the previous network tests did no roll-back of the sysctl.  Let me look into product methods to tune the sysctl, although I won't anticipate this being a requirement forever.  When this is fixed (https://github.com/distributed-system-analysis/pbench/issues/1238) we won't actually need to set the sysctl. (And thus no requirement for a privileged pods for Service to Service benchmarks then)

Reply:
Any settings that might affect other tests need to perform a roll-back I'd say.  Every test needs to try to be a good citizen and clean-up after itself.

Reply:
Ok It looks like I should be able to set this via a SecurityContext instead.  I also believe that this sysctl is namespaced so we have no worries on it affecting the http tests when run serially.  As far as I can tell it is also a safe sysctl so technically if we have test concurrency we could run them at the same time without any sysctl side affects but just test concurrency effects. 

https://docs.okd.io/latest/admin_guide/sysctls.html#safe-vs-unsafe-sysclts

Reply:
Also I absolutely agree that the tests should not leave any "side-affects" and reviews like this will help catch if that is the case, so thanks!

Reply:
Yes, this is one of the namespaced and 3 safe sysctls and I'm actually setting this already for HTTP tests here: https://github.com/openshift-scale/workloads/blob/master/workloads/files/workload-http-script-cm.yml#L862